testsuite/CUDA/cooperative_groups/pass/race_grid_group/kernel.cu

```diff
@@ -0,0 +1,19 @@
+//pass
```

Comment:
This kernel doesn't have a race. Can we move the file to a directory that doesn't include `race` in it's name? For example, `grid_barrier`.